src/main/java/org/generationcp/ibpworkbench/controller/UserProgramController.java

```diff
@@ -33,6 +33,11 @@
     @Resource
     private HttpServletRequest request;
 
+    //TODO:  this controller was implemented in IBP-4421 fixing an authentication issue.
+    //    After that in IBP-4397 was implemented the APIs to return the last project user selected and save the project selected by the user.
+    //    Also was implement ParamContext in AccountService and RouteAccessService
+    //    to set the crop and programUUID and recover the user authorities to validate the access.
+
```

Comment:
looks good :+1: , maybe one more line:
_remove after verifying that no other module depends on these values being in context info_